sample/src/main.js

```diff
@@ -1,8 +1,27 @@
+import Backend from 'i18next-xhr-backend';
+import {I18N} from 'aurelia-i18n';
+
 export function configure(aurelia) {
   aurelia.use
     .standardConfiguration()
     .developmentLogging()
-    .plugin('aurelia-validatejs');
+    .plugin('aurelia-i18n', (instance) => {
+      instance.i18next.use(Backend);
+
+      return instance.setup({
+        backend: {
+          loadPath: '/locales/{{lng}}/{{ns}}.json'
+        },
+        lng: 'nl'
+      });
+    })
+    .plugin('aurelia-validatejs', config => {
+      let i18n = aurelia.container.get(I18N);
+      config.useTranslation(error => {
+        let translation = i18n.tr(error.rule, error);
+        return translation !== error.rule ? translation : undefined;
```

Comment:
this is pretty ugly as you can't ask i18n if there is a translation for a key. When there isn't a translation for a key, it just returns the key